test/jdk/tools/jpackage/share/AddLauncherTest.java

```diff
@@ -200,6 +202,33 @@ public void testMainLauncherIsModular(boolean mainLauncherIsModular) {
         .applyTo(cmd);
 
         cmd.executeAndAssertHelloAppImageCreated();
+
+        // verify using each additional launchers cfg file that it has the
+        // main-jar/class or module/class
+
+        CfgFile cfg = cmd.readLauncherCfgFile("ModularAppLauncher");
+        String moduleValue = cfg.getValue("Application", "app.mainmodule");
+        String mainClass = null;
+        String classpath = null;
+        TKit.assertEquals(moduleValue, JavaAppDesc.parse(
+                modularAppDesc.toString()).setBundleFileName(null).toString(),
+                "app.mainmodule value in cfg file not as expected");
```

Comment:
I'd suggest to change comment to neutral "Check value of app.mainmodule in cfg file".